docs/source/system_reference_guide/oss_documentation/general_oss.ipynb

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
<p>&gt;...provides guidance on how NASA-funded projects and activities should share publications, data, and software</p><p>...</p><p> </p><p>I was thinking if it would be good to provide one example for each of these. It's always easier to understand from examples. However, since we have already added the original source please feel free to ignore it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>